lib/extensions/extension-helper.js

```diff
@@ -20,7 +19,47 @@
             var matched = keywordRegex.test(message.data.text);
             return matched;
         }
+    };
+
+function parseURL(url){
```

Comment:
Seeing as how you've added parseUrl to the extension_helper I'd want to see at least one or two specs covering this added to extension_helper_spec.js